src/services/schema.tsx

```diff
@@ -64,9 +72,8 @@ const getProperties = (obj) => {
  * class properties list.
  */
 const getQueryProperties = (className) => {
-  const VPropKeys = schemaDefn.schema.V.properties;
-  const classModel = schemaDefn.get(className);
-  return Object.values(classModel?.queryProperties ?? {})
+  const VPropKeys = schemaDefn.getProperties('V');
+  return Object.values(schemaDefn.queryableProperties(className) ?? {})
     .filter((prop) => !VPropKeys[prop.name]);
```

Comment:
Can you also specify 'prop''s type here (PropertyDefinition) so typescript wouldn't flag it as unknown?